lib/views/Table.js

```diff
@@ -53,6 +54,16 @@ Table.prototype.tabularise = function () {
     var data = this.data;
     var valueAttr = yAxis.key || this.moduleConfig['value-attribute'];
 
+    //sort the data if there is a sortby
+    if (sortBy) {
+      var sortOrder = this.moduleConfig['sort-order'] ?
+        this.moduleConfig['sort-order'] : 'descending';
```

Comment:
Do we know for sure that if 'sort-order' is absent, it will default to descending?